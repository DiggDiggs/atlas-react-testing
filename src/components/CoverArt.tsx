import React from "react";
import placeholderImage from "../assets/placeholder.svg";

const CoverArt: React.FC = () => {
  return (
    <div className="cover-art mb-6">
      <img
        src={placeholderImage}
        alt="Placeholder Cover Art Image"
        className="h-96 rounded-md"
      />
    </div>
  );
};

export default CoverArt;
