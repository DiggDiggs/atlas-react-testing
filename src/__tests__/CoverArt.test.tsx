import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import CoverArt from "../components/CoverArt";

describe("CoverArt Component", () => {
  it("should match the snapshot", () => {
    const { asFragment } = render(<CoverArt />);
    expect(asFragment()).toMatchSnapshot();
  });
});
