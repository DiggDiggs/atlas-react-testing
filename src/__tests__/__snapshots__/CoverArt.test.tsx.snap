// Jest Snapshot v1, https://goo.gl/fbAQLP
exports[`CoverArt Component should match the snapshot 1`] = `
<div
  class="cover-art mb-6"
>
  <img
    alt="Placeholder Cover Art Image"
    class="rounded-md h-96"
    src="/path/to/placeholder.svg"
  />
</div>
`;
